index.md

```diff
@@ -0,0 +1,24 @@
+# Django Paris
+
+Official website for Django Paris meetups.
+Find more information about next meetings by visiting the [Django Paris meetup page](https://www.meetup.com/fr-FR/django-meetup-paris/).
+
+## About
+
+🇫🇷 / 🇬🇧
+Hi everyone,
+
+The purpose of the group is to gather djangonauts around Paris to exchange and share knowledge, and of course, meet each other !
+
+No need to be an expert on Django, everyone is welcome.
+
+
+Bonjour à tous,
+
+Ce groupe à pour but de réunir les djangonautes autour de Paris afin d'échanger et partager des connaissances et bien sur, se rencontrer !
+
+Pas besoin d'être un expert en Django, tout le monde est le bienvenu.
```

Comment:
Should we translate everything? I don't know if it's will be good to have a link with all pages translated in one language, what do you think?

Reply:
Right, maybe we can leave it as is for now as we do not have much content and add different links with the EN/FR version later?